nautobot_ssot/integrations/aci/jobs.py

```diff
@@ -52,9 +51,7 @@ class Meta:  # pylint: disable=too-few-public-methods
     def __init__(self):
         """Initialize ExampleYAMLDataSource."""
         super().__init__()
-        self.diffsync_flags = (
-            self.diffsync_flags | DiffSyncFlags.SKIP_UNMATCHED_DST  # pylint: disable=unsupported-binary-operation
-        )
+        self.diffsync_flags = (self.diffsync_flags,)
```

Comment:
Are we just removing the SKIP_UNMATCHED_DST DiffSync flag here? if so, we can just remove the dunder init.